### PR TITLE
Added SHA256 & SHA1 application certificate fingerprints to README

### PR DESCRIPTION
**Pull request by DanyPM**
## Purpose / Description
Add the app certificate fingerprint to the README file, enabling apps and users to check the integrity of apk installed from github.

## Approach
The main purpose is to enable subsequent verification of the app through [AppVerifier](https://github.com/soupslurpr/AppVerifier), including adding the fingerprint to its internal database.

## How Has This Been Tested?

Not modified application code: only modifies the README file.

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [ ] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,11 @@ Install
 
 </div>
 
+Signing certificate fingerprint to [verify](https://developer.android.com/studio/command-line/apksigner#usage-verify) the APK:
+```
+SHA-256: 2071534f0f4b5e54ae952dd275d70da6e3459ee69909d2ab1b4843c4c5b21a45
```
Comment (david-allison) [inline README.md:59]: Is there a reason for excluding the SHA-1? [`f24e06a3657b190a12671100402df32d7b9b3d36`] 

There are search results from multiple sites for this, which adds to credibility

**Review by david-allison — COMMENTED** [review]
Looks great, SHA verified. Thanks!!

One question: implementer's choice on the outcome. 

**Maintainers**: Treat this comment as an 'approve' regardless of outcome

File: README.md (hunk 1 above)
Comment (DanyPM) [reply README.md:59]: SHA-1 added

**Review by david-allison — APPROVED** [review]
Thanks so much for the quick turnaround!

**Comment by AnClicks** [comment]
@david-allison can you  assign me this issue?

**Comment by david-allison** [comment]
@AnClicks this is an open pull request

**Pull request merged**